libs/libc/stream/lib_blkoutstream.c

```diff
@@ -127,6 +127,16 @@ static int blkoutstream_puts(FAR struct lib_outstream_s *this,
   return len;
 }
 
+/****************************************************************************
+ * Name: blkoutstream_putc
+ ****************************************************************************/
+
+static void blkoutstream_putc(FAR struct lib_outstream_s *this, int ch)
```

Comment:
Why not

```suggestion
static void blkoutstream_put(FAR struct lib_outstream_s *this, int ch)
```
?

Reply:
Because other C code used the name of putc, such as memoutstream_putc, lowoutstream_putc

Reply:
I named them putc to be consistent.